js/exporter/jspdf/export.js

```diff
@@ -56,10 +56,16 @@ export const Export = {
             jsPDFDocument,
             autoTableOptions,
             component,
+            customizeCell,
             keepColumnWidths,
             selectedRowsOnly
         } = options;
         const dataProvider = component.getDataProvider(selectedRowsOnly);
+        const customizeCellAction = (args) => {
```

Comment:
I think we don't need additional wrapper function for call of customizeCell.

Reply:
fixed